cli/cmd/generate_aws.go

```diff
@@ -29,47 +27,24 @@ var (
 	QuestionSubAccountRegion            = "What region should be used for this account?"
 	QuestionSubAccountAddMore           = "Add another AWS account?"
 	QuestionSubAccountReplace           = "Currently configured AWS subaccounts: %s, replace?"
-	QuestionConfigAdvanced              = "Configure advanced integration options?"
-	QuestionAnotherAdvancedOpt          = "Configure another advanced integration option"
-	QuestionCustomizeOutputLocation     = "Provide the location for the output to be written:"
+	QuestionAwsConfigAdvanced           = "Configure advanced integration options?"
+	QuestionAwsAnotherAdvancedOpt       = "Configure another advanced integration option"
+	QuestionAwsCustomizeOutputLocation  = "Provide the location for the output to be written:"
 
 	// select options
-	AdvancedOptDone        = "Done"
+	AwsAdvancedOptDone     = "Done"
 	AdvancedOptCloudTrail  = "Additional Cloudtrail options"
 	AdvancedOptIamRole     = "Configure Lacework integration with an existing IAM role"
 	AdvancedOptAwsAccounts = "Add additional AWS Accounts to Lacework"
-	AdvancedOptLocation    = "Customize output location"
+	AwsAdvancedOptLocation = "Customize output location"
 
-	// original source: https://regex101.com/r/pOfxYN/1
+	// AwsArnRegex original source: https://regex101.com/r/pOfxYN/1
 	AwsArnRegex = `^arn:(?P<Partition>[^:\n]*):(?P<Service>[^:\n]*):(?P<Region>[^:\n]*):(?P<AccountID>[^:\n]*):(?P<Ignore>(?P<ResourceType>[^:\/\n]*)[:\/])?(?P<Resource>.*)$`
-	// regex used for validating region input; note intentionally does not match gov cloud
+	// AwsRegionRegex regex used for validating region input; note intentionally does not match gov cloud
 	AwsRegionRegex  = `(us|ap|ca|cn|eu|sa)-(central|(north|south)?(east|west)?)-\d`
```

Comment:
Some thoughts for future, not necessarily this PR.
We have individual unit tests for each of these. I'm wondering if we can ensure all new regex is tested by using a custom type, create a list of those types and have a test run regexp.MustCompile.

type regex string

func (r regex) ToString() string {
	return ""
}

var (
AwsArnRegex regex = `([A-Za-z_0-9-]+`
AwsRegionRegex  regex = `([A-Za-z_0-9-]+`
AwsProfileRegex regex = `([A-Za-z_0-9-]+)`
)

func TestLwGenerateRegexValidators(t *testing.T) {
	for _, r := range allMyRegex {
		regexp.MustCompile(r)
	}
}